<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGameDialog</class>
 <widget class="QDialog" name="NewGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Game</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>&amp;White</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QRadioButton" name="whiteHumanRadioButton">
          <property name="text">
           <string>&amp;Human</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_4</string>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QRadioButton" name="whiteOnBoardRadioButton">
            <property name="text">
             <string>Play on &amp;Board</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="whiteInAppRadioButton">
            <property name="text">
             <string>Play in &amp;App</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="whiteAiRadioButton">
          <property name="text">
           <string>&amp;AI</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_4</string>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Difficulty:</string>
            </property>
            <property name="buddy">
             <cstring>whiteAiDifficultySlider</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="whiteAiDifficultySlider">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>600</number>
            </property>
            <property name="maximum">
             <number>1200</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="pageStep">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>&amp;Black</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QRadioButton" name="blackHumanRadioButton">
          <property name="text">
           <string>&amp;Human</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_3</string>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QRadioButton" name="blackOnBoardRadioButton">
            <property name="text">
             <string>Play on &amp;Board</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="blackInAppRadioButton">
            <property name="text">
             <string>Play in &amp;App</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="blackAiRadioButton">
          <property name="text">
           <string>&amp;AI</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_3</string>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Difficulty:</string>
            </property>
            <property name="buddy">
             <cstring>blackAiDifficultySlider</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="blackAiDifficultySlider">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>600</number>
            </property>
            <property name="maximum">
             <number>1200</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="pageStep">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewGameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewGameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>whiteHumanRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>whiteOnBoardRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>whiteHumanRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>whiteInAppRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>whiteAiRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>whiteAiDifficultySlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>whiteAiRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blackHumanRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>blackOnBoardRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blackHumanRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>blackInAppRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blackAiRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>blackAiDifficultySlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blackAiRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_4"/>
 </buttongroups>
</ui>
